modules/xfem/doc/config.yml

```diff
@@ -15,7 +15,7 @@ Extensions:
         executable: ${MOOSE_DIR}/modules/xfem
         hide:
             framework: !include framework/doc/hidden.yml
-            xfem: !include modules/xfem/doc/hidden.yml
+            #xfem: !include modules/xfem/doc/hidden.yml
```

Comment:
Can you just include this anyway even though it's empty?

Reply:
I followed the convention set forth by geochemistry.  It gave me an error previously on this